### PR TITLE
added some styling

### PR DESCRIPTION
**Pull request by developerJuiceLozzoc**
-videoplayer does a 2 decimal digits only stirng
-dashboard isnt entirely new or returning prompt.

-now has a banner to keep user information and signout.

### CODE REVIEW

File: client/src/components/Banner.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Button} from 'lucid-ui';
+import {Link} from "react-router-dom";
+
+
+
+function signOut(){
```
Comment (developerJuiceLozzoc) [inline client/src/components/Banner.js]: videopage js code for signing out was moved here in banner

Comment (sammysamx20) [reply client/src/components/Banner.js]: yep just saw that

Comment (developerJuiceLozzoc) [reply client/src/components/Banner.js]: im not sure what we should do about line 9, perhaps another solution?

File: client/src/components/Banner.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  localStorage.setItem("username", "");
+  window.location.href= "/login";
```
Comment (sammysamx20) [inline client/src/components/Banner.js]: Yeah dont do this please, This actually doesnt push on to the state of react router 

```    this.props.history.push('/')	```

**Comment by sammysamx20** [comment]
Also I noticed you're using React Hooks Syntax now and not the React class any more. We should stick to one syntax if possible.

File: client/src/components/VideoPage.js [hunk 3]
```diff
@@ -61,13 +61,7 @@ class VideoPage extends Component{
       chosenLabel: this.state.labels[value]
     });
   }
-  signOut(){
-    this.props.setGlobalUsername("");
-
-    localStorage.setItem("username", "");
-    this.props.history.push('/')
```
Comment (developerJuiceLozzoc) [inline client/src/components/VideoPage.js:68]: the path should be "/" and not "/login"

**Pull request merged**